src/popup/components/IconWrapper.vue

```diff
@@ -43,6 +44,7 @@ export default defineComponent({
   },
   props: {
     icon: { type: Object as PropType<Component>, default: null },
+    isFull: { type: Boolean, default: false },
```

Comment:
```suggestion
    isFull: Boolean,
```
`false` is always the default value for booleans.